skynet/modules/ttt/processor.py

```diff
@@ -153,6 +155,7 @@ async def process_text(model: BaseChatModel, payload: DocumentPayload) -> str:
 
     prompt = ChatPromptTemplate(
         [
+            ('system', set_response_language(payload.preferred_locale)),
```

Comment:
I would leave it out from process_text, the user should take care of any prompting.

Reply:
Nah, they get full prompt control, so the user is on the driver seat...